powerdns_recursor/datadog_checks/powerdns_recursor/powerdns_recursor.py

```diff
@@ -126,6 +126,27 @@ def check(self, instance):
                     self.gauge('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
                 elif stat['name'] in PowerDNSRecursorCheck.RATE_METRICS_V4:
                     self.rate('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
+        self._collect_metadata(config)
```

Comment:
(Nit) Extra blank line for readability

```suggestion

        self._collect_metadata(config)
```